data/card_catalogue/unique-1.json

```diff
@@ -169,15 +169,15 @@
     }
   },
   {
-    "title": "The Ultimatum",
+    "title": "The Gambler",
     "upgrade": null,
-    "tier": "blue",
+    "tier": "green",
     "details": {
       "rules": {
-        "active": "Flip a coin in place of your next roll. Heads for a critical success, tails for a critical failure.",
-        "passive": null
+        "active": "Summon a rigged pair of dice.",
+        "passive": "Whenever you would roll a d20, flip a coin instead. Heads for a critical success, tails for a critical failure." 
```

Comment:
This passive is incredibly disruptive to the game, but I think it'll be pretty funny. The active doesn't get much from being saved, so the player should be able to turn this off pretty easily